charts/kube-prometheus-stack/ci/03-non-defaults-values.yaml

```diff
@@ -28,6 +28,31 @@ alertmanager:
       logLevel: {{ print "debug" | quote }}
 
 prometheus:
+  gatewayAPI:
```

Comment:
I think this values file does not match the new syntax right?

Reply:
Thanks!